includes/class-branding.php

```diff
@@ -122,9 +122,9 @@ public function output_admin_notice() {
 			case 'update-core-network':
 				$message = sprintf(
 					/* translators: 1: The name of the plugin, 2: The documentation URL. */
-					__( 'Your WordPress, plugin, theme and translation updates are now powered by <strong>%1$s</strong>. <a href="%2$s">Learn more</a>', 'AspireUpdate' ),
-					'AspireUpdate',
-					__( 'https://docs.aspirepress.org/aspireupdate/', 'AspireUpdate' )
+					__( 'Your WordPress, plugin, theme and translation updates are now powered by <strong>%1$s</strong>. <a href="%2$s">Learn more</a>', 'aspireupdate' ),
+					'aspireupdate',
```

Comment:
This one should remain -> It's the name of the plugin being inserted into `%1$s` in the `sprintf()` call.

Reply:
fixed